### ISSUE TITLE
Add Duplicate Widget Name Validation to the Package Validator

### ISSUE THREAD

**Issue by TheConnMan** [metricly/metricly-cli#42]
No two widgets in the same dashboard are allowed to have the same name.

**Comment by bspindler** [comment]
Yes!  Finally something to warrant a release with our new awesome README :)

**Comment by Grumblesmith** [comment]
Is it true that widget names must be unique? Dashboard `42079a9a-097c-4326-a119-40c7b62bd653` makes me think it's not.
